OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -822,5 +822,46 @@ public void TestIsFeatureExperimentReturnsTrueForExperimentThatBelongsToAFeature
 
             Assert.True(typedConfig.IsFeatureExperiment(experiment.Id));
         }
+        
+        [Test]
+        public void TestRolloutWithEmptyStringRolloutIdFromConfigFile()
+        {
+            var projectConfig = DatafileProjectConfig.Create(TestData.EmptyRolloutDatafile, new NoOpLogger(), new NoOpErrorHandler());
+            Assert.IsNotNull(projectConfig);
+            var featureFlag = projectConfig.FeatureKeyMap["empty_rollout_id"];
+            
+            var rollout = projectConfig.GetRolloutFromId(featureFlag.RolloutId);
+
+            // OptlyConfig.OptimizelyConfigService.GetDeliveryRules L362
```

Comment:
Please remove this comment specifically Line number, that can be change in future.